src/components/displayTable/Row.tsx

```diff
@@ -46,7 +60,30 @@ const TableRow = ({ row, index }: { row: RowType; index: number }) => {
       {/* Collapsible Container */}
       {isExpand && (
         <tr className="bg-slate-100">
-          <td colSpan={9}>Helllo world</td>
+          <td colSpan={9}>
+            <table style={{ width: "100%" }}>
+              <tbody>
+                <tr>
+                  <td>Test_type: {type}</td>
```

Comment:
this should be a configurable list via a separate json file. you can loop over them. keep a note of this and make an issue. 

json file will have key_name and key pair. 
{"Test Takers Count": "testTakers", "Test Format": "format"} 

and you can write a loop here -- that iterates over this json file. it displays the `key_name` in header, and db_row[key] as column value. 

Current code is not easily configurable. With this json file, we just have to make a change in json and it will handle everything. Also, current code is repetitive wherein each column/row can be easily iterated over.